django/core/checks/model_checks.py

```diff
@@ -31,35 +31,127 @@ def check_all_models(app_configs=None, **kwargs):
     return errors
 
 
-@register(Tags.models, Tags.signals)
-def check_model_signals(app_configs=None, **kwargs):
+def _check_lazy_references(app_registry, ignore=None):
```

Comment:
I think the norm is to use `apps` instead of `app_registry` to designate `Apps()` instances around the codebase.